dataactcore/read_config.py

```diff
@@ -97,49 +98,45 @@
 normpath(broker_files)
 if broker_files[-1] != os.path.sep:
     broker_files += os.path.sep
-CONFIG_DICT["CONFIG_BROKER"]['broker_files'] = broker_files
+CONFIG_BROKER['broker_files'] = broker_files
 
 # normalize logging path, if given
-log_path = CONFIG_DICT["CONFIG_LOGGING"]['log_files']
+log_path = CONFIG_LOGGING['log_files']
 if log_path:
-    CONFIG_DICT["CONFIG_LOGGING"]['log_files'] = normpath(log_path)
+    CONFIG_LOGGING['log_files'] = normpath(log_path)
 
 # we don't want http:// or ports in the host variables
-CONFIG_DICT["CONFIG_SERVICES"]['broker_api_host'] = re.sub(
-    'http://|:(.*)', '', CONFIG_DICT["CONFIG_SERVICES"]['broker_api_host'])
-CONFIG_DICT["CONFIG_SERVICES"]['validator_host'] = re.sub(
-    'http://|:(.*)', '', CONFIG_DICT["CONFIG_SERVICES"]['validator_host'])
+CONFIG_SERVICES['broker_api_host'] = re.sub(
+    'http://|:(.*)', '', CONFIG_SERVICES['broker_api_host'])
+CONFIG_SERVICES['validator_host'] = re.sub(
+    'http://|:(.*)', '', CONFIG_SERVICES['validator_host'])
 # if hosts in config file are set to 0.0.0.0, override to
 # 127.0.0.1 for cross-platform compatibility
-if CONFIG_DICT["CONFIG_SERVICES"]['broker_api_host'] == '0.0.0.0':
-    CONFIG_DICT["CONFIG_SERVICES"]['broker_api_host'] = '127.0.0.1'
-if CONFIG_DICT["CONFIG_SERVICES"]['validator_host'] == '0.0.0.0':
-    CONFIG_DICT["CONFIG_SERVICES"]['validator_host'] = '127.0.0.1'
-if CONFIG_DICT["CONFIG_DB"]['dynamo_host'] == '0.0.0.0':
-    CONFIG_DICT["CONFIG_DB"]['dynamo_host'] = '127.0.0.1'
-
-if CONFIG_DICT["CONFIG_SERVICES"]["broker_api_port"] == 443:
+if CONFIG_SERVICES['broker_api_host'] == '0.0.0.0':
+    CONFIG_SERVICES['broker_api_host'] = '127.0.0.1'
+if CONFIG_SERVICES['validator_host'] == '0.0.0.0':
+    CONFIG_SERVICES['validator_host'] = '127.0.0.1'
+if CONFIG_DB['dynamo_host'] == '0.0.0.0':
+    CONFIG_DB['dynamo_host'] = '127.0.0.1'
+
+if CONFIG_SERVICES["broker_api_port"] == 443:
     # Use https
-    CONFIG_DICT["CONFIG_SERVICES"]["protocol"] = "https"
+    CONFIG_SERVICES["protocol"] = "https"
 else:
-    CONFIG_DICT["CONFIG_SERVICES"]["protocol"] = "http"
+    CONFIG_SERVICES["protocol"] = "http"
 
 # Log some values from config
 log_message = ""
-if "values_to_log" in CONFIG_DICT["CONFIG_LOGGING"]:
+if "values_to_log" in CONFIG_LOGGING:
     # If no values specified, don't do logging
-    for category_yaml_name in CONFIG_DICT["CONFIG_LOGGING"]["values_to_log"]:
-        category_name = CONFIG_CATEGORIES[category_yaml_name]
-        category = CONFIG_DICT[category_name]
-        category_message = "### {}".format(category_name)
-        for key in CONFIG_DICT["CONFIG_LOGGING"]["values_to_log"][category_yaml_name]:
+    for category_yaml_name in CONFIG_LOGGING["values_to_log"]:
+        category = CONFIG_CATEGORIES[category_yaml_name]
+        category = category
```

Comment:
I'd like to use that excuse, but sadly that was actually me manually replacing locals()[category_name] with the new category variable from line 133.  I just didn't actually look at the left side of the equation.  Removing that now.